app/models/mixins/read_only_mixin.rb

```diff
@@ -0,0 +1,17 @@
+module ReadOnlyMixin
+  extend ActiveSupport::Concern
+
+  included do
+    before_destroy    :reject_if_read_only
```

Comment:
Minor but some extra padding here (assume was copied from a model where this was lined up right)